tests/core_tests/test_pipelineconfig_factory.py

```diff
@@ -400,7 +400,7 @@ def test_get_pipeline_configs(self):
             ]
         )
         self.assertEqual(
-            self._sg_data["local_storages"], [self._remove_items(self.primary_storage, "__retired")]
+            self._sg_data["local_storages"], [self._remove_items(self.primary_storage, ["__retired", "created_at", "updated_at"])]
```

Comment:
I modified some of the test data so that creation date is injected in mockgun objects. We'll have to update mockgun so this becomes standard.